pkg/operator/resource/resourceapply/core.go

```diff
@@ -88,14 +88,6 @@ func ApplySecret(ctx context.Context, client coreclientv1.SecretsGetter, recorde
 	return applySecretImproved(ctx, client, recorder, required, noCache, false)
 }
 
-// ApplySecretDoNotUse is depreated and will be removed
-// Deprecated: DO NOT USE, it is intended as a short term hack for a very specific use case,
-// and it works in tandem with a particular carry patch applied to the openshift kube-apiserver.
-// Use ApplySecret instead.
-func ApplySecretDoNotUse(ctx context.Context, client coreclientv1.SecretsGetter, recorder events.Recorder, required *corev1.Secret) (*corev1.Secret, bool, error) {
```

Comment:
The original [PR](https://github.com/openshift/library-go/pull/1705/files#diff-81c137f4d13fefcae5735d1cf0e4aaaf14c96b3e4931c1fd1a242ae1a25a5489) introduced more changes. Not all were remove. Please open a new PR that reverts https://github.com/openshift/library-go/pull/1705.